mcs/class/Mono.Posix/Assembly/AssemblyInfo.cs

```diff
@@ -36,7 +36,12 @@
 
 [assembly: AssemblyVersion (Consts.FxVersion)]
 
+#if MONO_POSIX_NETSTANDARD_BUILD
+[assembly: AssemblyTitle("Mono.Posix.NETStandard.dll")]
```

Comment:
Why not version it 1.0.0.0 when it's a new assembly and use [OpenKey.snk](https://github.com/mono/mono/blob/master/mcs/class/Open.snk) like we do for other new netstandard assemblies ?